File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] result=new int[2];
        int start=0, end=numbers.length-1;
        while(numbers[start] + numbers[end] != target)
        {
            if(numbers[start] + numbers[end] < target)
            {
                start++;
            }
            else
                end--;
        }
        start++;
        end++;
        result[0]=start;
        result[1]=end;
        return result;
    }
}

File: Data Structures & Algorithms/generate-parentheses/submission-0.py
class Solution:
    def generateParenthesis(self, n: int) -> List[str]:
        result=[]
        def helper(s,r, l):
            if (2 * n) == len(s):
                result.append(s)
                return
            if l < n:
                helper(s + "(", r, l + 1)
            if  r < l:
                helper(s + ")", r + 1, l)    
        helper("", 0, 0)
        return result    